modules/oe_webtools_laco_widget/README.md

```diff
@@ -0,0 +1,51 @@
+# OpenEuropa Webtools Laco Widget
+
+Provides a widget that integrates with the Webtools Laco service and which checks if a 
+link to o page or document is available in other translations.
+
+### How it works
+
+This module simply outputs the Webtools Laco Widget on every page and provides default
+configurations for it. More information on this service can be found [here](https://webgate.ec.europa.eu/fpfis/wikis/pages/viewpage.action?spaceKey=webtools&title=Language+Coverage).
+
+
+### Configuration
+
+The default widget that is being output is the following:
+
+```
+{
+    "service"    : "laco",
+    "include"    : "#page-wrapper",
```

Comment:
Use `body` so it can work by default anywhere regardless of which theme we are using.